app/jobs/parse_csv_import_job.rb

```diff
@@ -1,17 +1,32 @@
 require 'csv'
 require 'open-uri'
+require 'charlock_holmes/string'
 
 class ParseCsvImportJob < ApplicationJob
+  attr_accessor :import
+
   def perform(import_id)
-    import = Import.find_by id: import_id
-    return unless import
+    @import = Import.find_by id: import_id
+    return unless @import
+
+    create_raw_inputs
+    @import.transform
```

Comment:
`transform` is more descriptive up here in `perform`. Feels less lost in the details.